libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -181,9 +187,12 @@ uint16_t SRV_Channel::get_limit_pwm(LimitValue limit) const
     }
 }
 
-// return true if function is for a multicopter motor
+// return true if function is for a motor of any sort
 bool SRV_Channel::is_motor(SRV_Channel::Aux_servo_function_t function)
 {
-    return ((function >= SRV_Channel::k_motor1 && function <= SRV_Channel::k_motor8) ||
```

Comment:
As discussed on the dev call, it would probably be best to revert this part of the change and instead create a new function to check if a function is for a "spinny bit".